components/layout/dom_traversal.rs

```diff
@@ -84,6 +84,8 @@ pub(super) enum Contents {
     /// Example: an `<img src=…>` element.
     /// <https://drafts.csswg.org/css2/conform.html#replaced-element>
     Replaced(ReplacedContents),
+    /// A video element with widget.
+    ReplacedWithWidget(NonReplacedContents),
```

Comment:
AS an option - may be use `ReplacedWithWidget(ReplacedContents, NonReplacedContents),`, so later it will be possible to use this information in layout Box/Fragment construction?